workflows/scRNAseq/pseudobulk-worflow-decoupler-edger/pseudo-bulk_edgeR.ga

```diff
@@ -0,0 +1,1023 @@
+{
+    "a_galaxy_workflow": "true",
+    "annotation": "Pseudobulk",
+    "comments": [],
+    "creator": [
+        {
+            "class": "Person",
+            "identifier": "0000-0002-5857-1477",
+            "name": "Diana Chiang Jurado"
+        },
+        {
+            "class": "Person",
+            "identifier": "0000-0002-5192-126X",
+            "name": "Pavankumar Videm"
+        },
+        {
+            "class": "Person",
+            "identifier": "0000-0002-9856-1679",
+            "name": "Pablo Moreno"
+        }
+    ],
+    "format-version": "0.1",
+    "license": "CC-BY-4.0",
+    "release": "0.1",
+    "name": "pseudo-bulk_edgeR",
```

Comment:
```suggestion
    "name": "Differential gene expression for single-cell data using pseudo-bulk counts with edgeR",
```
Or something like this ? That's what the users will see in galaxy and https://iwc.galaxyproject.org/